charts/k6-operator/README.md

```diff
@@ -1,6 +1,6 @@
 # k6-operator
 
-![Version: 3.1.0](https://img.shields.io/badge/Version-3.1.0-informational?style=flat-square) ![AppVersion: 0.0.11](https://img.shields.io/badge/AppVersion-0.0.11-informational?style=flat-square)
+![Version: 3.3.0](https://img.shields.io/badge/Version-3.3.0-informational?style=flat-square) ![AppVersion: 0.0.11](https://img.shields.io/badge/AppVersion-0.0.11-informational?style=flat-square)
```

Comment:
Please update `charts/k6-operator/Chart.yaml` instead and run `helm-docs` command. This Readme is not supposed to be edited manually.

Reply:
@llaszkie, thank you but you need to git commit the `charts/k6-operator/Chart.yaml` too - it is a tracked file :sweat_smile: